examples/crud/main.go

```diff
@@ -38,16 +38,16 @@ func main() {
 
 	// List all sobjects
 	queryParams := sdkms.ListSobjectsParams{
-		Sort: sdkms.SobjectSort{
+		Sort: &sdkms.SobjectSort{
 			ByName: &sdkms.SobjectSortByName{},
 		},
 	}
```

Comment:
It seems the `ListSobjectsResponse` type does not compatible without Metadata, so you need to ensure response always comes with Metadata by requiring it. At default: back-end response a `[]Sobject` which is not compatible with  `ListSobjectsResponse`
```suggestion
	withMetadata := true
	queryParams := sdkms.ListSobjectsParams{
		Sort: &sdkms.SobjectSort{
			ByName: &sdkms.SobjectSortByName{},
		},
		WithMetadata: &withMetadata,
	}
```

Reply:
To completely resolve this, I suggest you to add something similar to what you did for `AuditLogTime` in `support.go` for `ListSobjectsResponse`